src/api-reference/expense/allocations/expense-allocations.markdown

```diff
@@ -3,7 +3,7 @@ title: Allocations
 layout: reference
 ---
 
-This resource can be used to retrieve information about the allocations that are associated with an entry in an expense report. v3.0
+The SAP Concur Allocations API allows for the retrieval of allocation information as it relates to a Report ID, Entry ID, or Itemization ID. Using this API allows for an in-depth review of Expense Report Data and how that data has been allocated in SAP Concur. The Allocations Web Service allows for the programmatic gathering of details on how the Expense Report data was allocated by the report owner, making it ideal for Data Gathering, Expense Reporting, and Validations. This resource can be used to retrieve information about the allocations that are associated with an entry in an expense report. v3.0
```

Comment:
Remove the last sentence and the version. "Expense Report Data" shouldn't be capitalized. Change "Allocations Web Service" to "Allocations API"